### PR TITLE
Database schema

### PR DESCRIPTION
**Pull request by Amoodaa**
Did database schema and formatted as wanted
with the fake data
relates #64

### CODE REVIEW

**Comment by Fatmasiam** [comment]
@Amoodaa  @MaiUbeid   insert some fake data

File: server/database/config/dbbuild.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
+BEGIN;
+DROP TABLE IF EXISTS users, languages, channels, user_channel, user_interest, messages CASCADE;
+CREATE TABLE languages (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(50) NOT NULL
+);
```
Comment (Fatmasiam) [inline server/database/config/dbbuild.sql]: it's better to add a line between the table and the next table 
its make the file more readable

File: server/database/config/insertFakeData.sql [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  (3,4);
+
+INSERT INTO messages
+  (user_id, channel_id, content, sent_at)
+VALUES
+  (1, 1, 'hey', CURRENT_TIMESTAMP),
+  (1, 2, 'hey2', CURRENT_TIMESTAMP),
+  (1, 3, 'hey3', CURRENT_TIMESTAMP),
+  (2, 1, 'hey4', CURRENT_TIMESTAMP),
+  (1, 3, 'hey1', CURRENT_TIMESTAMP);
```
Comment (Fatmasiam) [inline server/database/config/insertFakeData.sql]: add new line

File: server/database/config/insertFakeData.sql [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+INSERT INTO user_interest
+  (interest_id, user_id)
+VALUES
+  (1,1),
+  (1,2),
+  (2,1),
+  (5,2),
+  (3,4),
+  (5,4);
+INSERT INTO user_channel
```
Comment (Fatmasiam) [inline server/database/config/insertFakeData.sql]: add line between the ` INSERT` statement and other

File: server/database/config/dbbuild.sql [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  avatar_path VARCHAR(255),
+  native_lang_id INTEGER NOT NULL REFERENCES languages (id),
+  learning_lang_id INTEGER NOT NULL REFERENCES languages (id)
+);
+
+CREATE TABLE channels (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL
+);
+
```
Comment (yosefalnajjarofficial) [inline server/database/config/dbbuild.sql]: I think it would be better if you put the user channel table right after the channel table

**Pull request merged**